src/types/user.ts

```diff
@@ -8,7 +8,7 @@ type RolesType = {
   member: boolean;
 };
 
-type MemberType = {
+type UserType = {
```

Comment:
- Is this type different from the one in `src/components/MembersSectionNew/types/MembersSection.type.tsx` can we use the type declared in this file everywhere a `UserType` is required?

Reply:
It's the same type only, Now I have re-used the existing type in wherever it is necessary 👍🏻 
Thanks for pointing out 🤝